FWCore/Framework/bin/cmsRun.cpp

```diff
@@ -166,7 +205,8 @@ int main(int argc, char* argv[]) {
           "Number of threads to use in job (0 is use all CPUs)")(
           kSizeOfStackForThreadCommandOpt,
           boost::program_options::value<unsigned int>(),
-          "Size of stack in KB to use for extra threads (0 is use system default size)")(kStrictOpt, "strict parsing");
+          "Size of stack in KB to use for extra threads (0 is use system default size)")(kStrictOpt, "strict parsing")(
+          kCmdCommandOpt, boost::program_options::value<std::string>(), "config passed in as string (config_file [python options] will be ignored)");
```

Comment:
I was following the `python` usage; from `python3 --help`:
```
-c cmd : program passed in as string (terminates option list)
```

Reply:
In fact, allowing `[python options]` with `-c [command line input]` conflicts with the below request for "an error message being shown if both `-c` and the positional configuration file", because it's ambiguous whether the first token after `[command line input]` is the config name or a python option. I'm also not sure why the use of `[python options]` with `-c` would ever be needed.